bot/exts/evergreen/emojis.py

```diff
@@ -0,0 +1,39 @@
+import textwrap
+
+from discord import Color, Embed, Emoji
+from discord.ext import commands
+
+from bot.utils.time import time_since
+
+
+class Emojis(commands.Cog):
+    """Has commands related to emojis."""
+
+    @commands.group(name="emoji", invoke_without_command=True)
+    async def emojis_group(self, ctx: commands.Context, emoji: Emoji) -> None:
+        """A group of commands related to emojis."""
+        # No parameters = same as invoking info subcommand
+        await ctx.invoke(self.info_command, emoji)
+
+    @emojis_group.command(name="info")
+    async def info_command(self, ctx: commands.Context, emoji: Emoji) -> None:
+        """Returns relevant information about a Discord Emoji."""
+        emoji_information = Embed(
+            title=f'Information about "{emoji.name}"',
+            description=textwrap.dedent(f"""
+                Name: {emoji.name}
+                Created: {time_since(emoji.created_at)}
```

Comment:
```suggestion
                Created: {time_since(emoji.created_at, precision="hours")}
```
Setting this to "days" could also be an option, if it feels to verbose.

Reply:
What's the output if the precision are hours?

Reply:
> What's the output if the precision are hours?  
And the emoji was created in less than an hour?

Reply:
The lowest detail that it'll include is hours. Like this:
`Created: 15 days and 3 hours ago `
Instead of:
`Created: 15 days, 3 hours, 36 minutes and 38 seconds ago`

Reply:
Ah, okay. I will accept this.